app/src/main/java/kr/co/connect/boostcamp/livewhere/firebase/BookmarkUserMapper.kt

```diff
@@ -0,0 +1,12 @@
+package kr.co.connect.boostcamp.livewhere.firebase
+
+import kr.co.connect.boostcamp.livewhere.model.BookmarkUser
+import kr.co.connect.boostcamp.livewhere.model.entity.BookmarkUserEntity
+
+class BookmarkUserMapper : FirebaseMapper<BookmarkUserEntity, BookmarkUser>() {
```

Comment:
`p3;` 내부 모듈에서 쓰는 것 뿐이라면 internal 키워드를 붙여주세요.
```suggestion
internal class BookmarkUserMapper : FirebaseMapper<BookmarkUserEntity, BookmarkUser>() {
```